src/lib/cache/subCache.cpp

```diff
@@ -735,7 +737,21 @@ void subCacheItemInsert
 
   if (stringFilterP != NULL)
   {
-    cSubP->expression.stringFilter = *stringFilterP;  // Object copy
+    std::string  errorString;
+
+    if (!cSubP->expression.stringFilter.fill(stringFilterP, &errorString))
+    {
+      LM_E(("Runtime Error (%s)", errorString.c_str()));
```

Comment:
I considered that when adding the LM_E ... 
And yes, you're right, might provoke over-logging.
Perhaps we should never log if we 'send errorStringP' to calling function